src/index.js

```diff
@@ -2,13 +2,19 @@ import './style.css';
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add your own name logo created by you at the top of the readme file to make your readme attractive and present you as a brand in the software industry 👍 to catch the recruiters' eyes 😊